envoy-control-tests/src/main/kotlin/pl/allegro/tech/servicemesh/envoycontrol/config/containers/HttpsEchoContainer.kt

```diff
@@ -1,11 +1,30 @@
 package pl.allegro.tech.servicemesh.envoycontrol.config.containers
 
+import com.fasterxml.jackson.databind.DeserializationFeature
+import com.fasterxml.jackson.databind.ObjectMapper
+import okhttp3.Response
+import org.assertj.core.api.Assertions
+import org.assertj.core.api.ObjectAssert
 import pl.allegro.tech.servicemesh.envoycontrol.config.BaseEnvoyTest
 
-class HttpsEchoContainer : SSLGenericContainer<HttpsEchoContainer>("mendhak/http-https-echo") {
+class HttpsEchoContainer : SSLGenericContainer<HttpsEchoContainer>("marcinfalkowski/http-https-echo") {
```

Comment:
Can you describe what changes your image include?

Reply:
Thanks for pointing it out, it turns out my changes are already merged to base image (I didn't expect it will be that fast), so I reverted the change.

The downside is the base image is not tagged, so we have to use image hash, otherwise the test will fail if there is an older version on local machine.

You can see the changes here: https://github.com/mendhak/docker-http-https-echo/pull/2